task/syft/0.1/syft.yaml

```
@@ -0,0 +1,32 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: syft
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, syft
+    tekton.dev/displayName: "syft"
```

```suggestion
    tekton.dev/displayName: syft
```

string doesn't need quoting

---

thanks